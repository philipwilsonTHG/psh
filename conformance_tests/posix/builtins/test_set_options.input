# Test set builtin options
echo "Testing set builtin"

# Test positional parameters
set -- arg1 arg2 arg3
echo $1
echo $2
echo $3
echo $#

# Test shell options
set -e
echo "errexit set"
set +e
echo "errexit unset"

# Test xtrace (suppress + output in golden file)
set -x 2>/dev/null
echo "xtrace test"
set +x 2>/dev/null

# Test showing all set options (limit output)
set -o 2>/dev/null | head -3