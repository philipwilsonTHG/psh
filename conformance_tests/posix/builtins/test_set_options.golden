Testing set builtin
arg1
arg2
arg3
3
errexit set
errexit unset
xtrace test
